cypress/integration/home-page.spec.js

```diff
@@ -438,4 +438,73 @@ describe('Home page', () => {
       })
     }
   )
+
+  context('When an operative is logged in', () => {
+    beforeEach(() => {
+      cy.loginWithOperativeRole()
+    })
+
+    it('Displays content in the header', () => {
+      cy.visit('/')
+
+      // Header component
+      cy.get('.lbh-header__service-name').contains('Repairs Hub')
+      cy.get('.lbh-header__title-link').should('have.attr', 'href', '/')
+
+      // Manage work orders link
+      cy.get('#manage')
+        .contains('Manage work orders')
+        .should('have.attr', 'href', '/')
+
+      // Search link
+      cy.get('#search')
+        .contains('Search')
+        .should('have.attr', 'href', '/search')
+
+      // Logout component
+      cy.get('#signout')
+        .contains('Sign out')
+        .should('have.attr', 'href', '/logout')
+    })
+
+    context('When they have work orders attached to them', () => {
+      beforeEach(() => {
+        cy.clock(new Date('June 11 2021 13:49:15Z'))
+
+        //Stub request with operative's work orders response
+        cy.intercept(
+          {
+            method: 'GET',
+            path: '/api/operatives/hu0001/workorders',
+          },
+          {
+            fixture: 'operatives/workOrders.json',
+          }
+        ).as('operativesWorkOrders')
+
+        cy.visit('/')
+        cy.wait('@operativesWorkOrders')
+      })
+
+      it('Displays work order appointments slots and priority', () => {
+        cy.get('.lbh-heading-h1').contains('Friday, 11 June')
+
+        cy.get('.lbh-list').within(() => {
+          cy.checkForAppointmentsDetails('08:00-13:00', '5 [N] NORMAL')
```

Comment:
We're not getting much benefit from an extracted function here. It's just a couple of content checks.

I'd prefer to have this all contained within the test and no separate function please.

I don't generally worry about duplicate code in test expectations. It's more valuable to see explicitly within each test what it is checking without having to jump into separate files. I know we have functions for user sign in, but those are more for making the setup easier.